src/PhenX.EntityFrameworkCore.BulkInsert/BulkInsertProviderBase.cs

```diff
@@ -147,15 +33,25 @@ public virtual async IAsyncEnumerable<T> BulkInsertReturnEntities<T>(
         OnConflictOptions? onConflict,
         [EnumeratorCancellation] CancellationToken ctk) where T : class
     {
+        using var activity = Telemetry.ActivitySource.StartActivity("BulkInsertReturnEntities");
+        activity?.AddTag("tableName", tableInfo.TableName);
+        activity?.AddTag("synchronous", true);
```

Comment:
I think you may want to use `sync` instead

Reply:
yes :D

Reply:
Fixed